web/app/routes/application.js

```diff
@@ -17,7 +17,6 @@ export default class ApplicationRoute extends Route {
   error(error) {
     if (error instanceof UnauthorizedError) {
       this.session.invalidate();
-      this.router.transitionTo("authenticate");
```

Comment:
Happens automatically via invalidate

Reply:
I've lost context for this... where is that route configured then for `invalidate`? 🤔

Reply:
Not 100%, but I think it automatically redirects to `/` which in our app is `/authenticated`, which has a `requireAuthentication` check that, if failed, redirects to `authenticate`.